### ISSUE TITLE
HKSV won’t work 

### ISSUE THREAD

**Issue by pillemats** [seydx/homebridge-camera-ui#656]
Hi

I am using the newest version of homebridge and camera ui
all works great but HKSV won’t work. I only can see the live mode and I set up stream and record.

Hopefully it is an config problem

I don’t know where is the problem

many thanks for help!
![F45D3E2B-D550-4DE4-8543-15DA9061A913](https://user-images.githubusercontent.com/26164977/193466211-deb320f4-ad32-4078-a6c2-2524a179432b.png)
![75039CD3-9217-454C-80D5-80220C050CDA](https://user-images.githubusercontent.com/26164977/193466217-33bd270e-1ed5-4843-a4c0-e6e60dd9f65b.png)
![ADF34699-A143-4B32-AD8D-D946F16211A2](https://user-images.githubusercontent.com/26164977/193466220-3b73dd13-35df-45b6-96a8-ca34bae87086.png)
![FD408E22-D6D3-41CC-8382-94BDE4BF3232](https://user-images.githubusercontent.com/26164977/193466222-bd41f870-0d7d-4400-ac8b-7c3a6cb310e0.png)
![EC87FAAC-9D1D-4DCC-AC42-45C51E760FFA](https://user-images.githubusercontent.com/26164977/193466225-bed396d1-88ce-4362-bc75-917286dc54b1.jpeg)

**Comment by jlg89** [comment]
Same here. I've tried a very simple HSV config, defining "audio": true and "maxbitrate": "2048", as well as the config below which AFAICT matches the documentation, and either way I get "The recording process was aborted by HSV with reason 'TIMEOUT'."

`
"cameras": [
    {
      "name": "Front Door",
      "manufacturer": "August Home",
      "model": "Doorbell Cam Pro",
      "serialNumber": "D2GWS0013U",
      "motion": true,
      "doorbell": false,
      "motionTimeout": 15,
      "videoConfig": {
        "source": "-i rtsp://admin:admin@192.168.77.7/stream",
        "stillImageSource": "-i rtsp://admin:admin@192.168.77.7/stream -vframes 1 -r 1",
        "maxWidth": 480,
        "maxHeight": 640,
        "stimeout": 0,
        "subSource": "-i rtsp://admin:admin@192.168.77.7/stream -r 10",
        "audio": true,
        "debug": true,
        "readRate": true
      },
      "mqtt": {
        "motionTopic": "",
        "motionResetTopic": "",
        "motionResetMessage": "",
        "motionMessage": ""
      },
      "prebufferLength": 4,
      "videoanalysis": {
        "active": true
      },
      "smtp": {
        "email": "Front Door"
      },
      "hsv": true,
      "hksvConfig": {
        "audio": true,
        "maxBitrate": "2048",
        "maxWidth": "1920",
        "maxHeight": "1080",
        "maxFPS": "30",
        "encoderOptions": "-g 120",
        "vcodec": "libx264",
        "acodec": "aac"
      },
      "prebuffering": true
    }
  ]
`

**Comment by AlbertEinslime** [comment]
Hello,
Having a similar issue. Somewhat new to Homebridge and camera.ui. Recently got a new Lorex system and have all cameras showing up fine in Home, and 3/4 of them work fine in terms of HKSV. However, one of them only recorded motion once and no longer saves any clips for me to view. I have attempted to toggle various settings on and off including stream/recording, but nothing seems to stick. I have also been unable to get audio to work on any of the cameras. Any help will be greatly appreciated. Thanks.

**Comment by karesake** [comment]
Got the same Error!

**Comment by fabvis** [comment]
same here !

**Comment by kakashysen** [comment]
Same issue here, the only way that works for me is using Scrypted, I've been using it for 10 days, and it works much better, it records everything in HKSV, with no issues, and pretty simple / easy to use

**Comment by pillemats** [comment]
Is there any solution ?

**Comment by jlg89** [comment]
At this point, if you want simple "homebridge-native" camera support, and HSV isn't important to you, use homebridge-camera-ffmpeg. If HSV is important, use Scrypted.

**Comment by tbelcher** [comment]
FWIW - HKSV hasn't't been working for me for the last couple of weeks, but yesterday I received two notifications mid-day.  None since then, but it makes me wonder what's changed.

**Comment by erokoder** [comment]
I configure yesterday, and HKSV don't work. I also have problem with video analysis, motion detection only one time detected, although it should be much more.

**Comment by jlg89** [comment]
For reliable HSV, I highly recommend setting up Scrypted in a Docker container.

**Comment by theapu** [comment]
HKSV motion detection and recording working only for the very first instance. Then no recording atall. But the logs show motion detected, video analysis etc. Same problme if I enable aws rekognition. For the very first time I got recording and notification with label but after that no recording and notification. No errors in logs. Very strange. I switched to scrypted for HKSV. I still use camera.ui for sending telegram notifications.
